Project/core/src/main/java/Model/MindSlip.java

```diff
@@ -0,0 +1,15 @@
+package Model;
+
+/**
+ * The MindSlip class represents a specific type of Token in the game.
+ * It overrides the toString method to provide a string representation
+ * of the MindSlip object.
+ */
+
+public class MindSlip extends Token {
+
+    @Override
+    public String toString() {
+        return "Mindslip{}";
+    }
+}
```

Comment:
this would be useful for future implementations as then a mindslip would be available for pickup on the board but right now a mindslip token shouldnt be displayed on the board.

Reply:
I choose to keep this file even it's not useful now.